### PR TITLE
Update cdm16 syntax highlighting

### PR DESCRIPTION
**Pull request by NikitaTrTr**
Update syntax highlighting of instructions, add cdm16 macros highlighting

### CODE REVIEW

File: vscode-cdm-extension/syntaxes/cdm16-assembly.tmLanguage.json [hunk 1]
```diff
@@ -51,7 +54,7 @@
...
 				"name": "storage.type.struct.cdm16-assembly",
-				"match": "\\b(beq|bz|bnne|bnnz|bne|bnz|bneq|bhs|bcs|bnlo|bncc|blo|bcc|bnhs|bncs|bmi|bnpl|bpl|bnmi|bvs|bnvc|bvc|bnvs|bhi|bnlc|bls|bnhi|bge|bnlt|blt|bnge|bgt|bnle|ble|bngt|br|banything|btrue|bnfalse|nop|bnanything|bntrue|bfalse|zero|trap|fexc|bad|halt|wait|ei|di|jsr|rti|pupc|popc|pusp|posp|pups|pops|es|ds|push|pop|ldi|jsrr|ldsp|stsp|ldps|stps|ldpc|stpc|shl|shr|shra|ror|rol|rcr|rcl|move|cmp|bit|ldw|ldb|ldsb|ldcw|ldcb|ldcsb|stw|stb|neg|not|sxt|scl|ldsw|ldsb|ldssb|stsw|stsb|ldi|add|cmp|int|push|addsp|jsr|ldw|ldb|ldcw|ldcb|ldcsb|stw|stb|and|or|xor|bic|add|adc|sub|sbc)\\b"
+				"match": "\\b(beq|bz|bnne|bnnz|bne|bnz|bneq|bhs|bcs|bnlo|bncc|blo|bcc|bnhs|bncs|bmi|bnpl|bpl|bnmi|bvs|bnvc|bvc|bnvs|bhi|bnlc|bls|bnhi|bge|bnlt|blt|bnge|bgt|bnle|ble|bngt|br|banything|btrue|bnfalse|nop|bnanything|bntrue|bfalse|zero|trap|fexc|bad|halt|wait|ei|di|jsr|rti|pupc|popc|pusp|posp|pups|pops|es|ds|push|pop|ldi|jsrr|ldsp|stsp|ldps|stps|ldpc|stpc|shl|shr|shra|ror|rol|rcr|rcl|move|cmp|bit|ldw|ldb|ldsb|lcw|lcb|lcsb|stw|stb|neg|not|sxt|scl|lsw|lsb|lssb|ssw|ssb|ldi|add|cmp|int|push|addsp|jsr|ldw|ldb|lcw|lcb|lcsb|stw|stb|and|or|xor|bic|add|addc|subc)\\b"
```
Comment (Intelix8996) [inline vscode-cdm-extension/syntaxes/cdm16-assembly.tmLanguage.json:57]: Remove dublicates

File: vscode-cdm-extension/syntaxes/cdm16-assembly.tmLanguage.json [hunk 2]
```diff
@@ -65,6 +68,12 @@
                 "end": "\\*/"
 			}]
 		},
+		"macros": {
+			"patterns": [{
+				"name": "keyword.control.instruction.cdm8-assembly",
+				"match": "\\b(run|else|if|fi|is|gt|lt|le|ge|mi|pl|eq|ne|z|nz|cs|cc|vs|vc|hi|lo|hs|ls|continue|wend|until|while|save|restore|stays|true|break|tst|clr|do|then|unique|first_item|item|last_item|bnls|rts|bis|inc|dec|ld|st|ldc|db|dw)\\b"
```
Comment (Intelix8996) [inline vscode-cdm-extension/syntaxes/cdm16-assembly.tmLanguage.json]: Remove `run`

**Review by Intelix8996 — CHANGES_REQUESTED** [review]
Need to resolve conversations

File: vscode-cdm-extension/syntaxes/cdm16-assembly.tmLanguage.json (hunk 1 above)
Comment (Intelix8996) [reply vscode-cdm-extension/syntaxes/cdm16-assembly.tmLanguage.json:57]: `sub` is missing

Comment (Intelix8996) [reply vscode-cdm-extension/syntaxes/cdm16-assembly.tmLanguage.json:57]: Remove `zero|trap|fexc|bad`

Comment (Intelix8996) [reply vscode-cdm-extension/syntaxes/cdm16-assembly.tmLanguage.json:57]: Remove `es|ds`

File: vscode-cdm-extension/syntaxes/cdm16-assembly.tmLanguage.json (hunk 2 above)
Comment (Intelix8996) [reply vscode-cdm-extension/syntaxes/cdm16-assembly.tmLanguage.json]: Move `db` and `dw` to `directives`

**Comment by Intelix8996** [comment]
Add `ext` to instructions

File: vscode-cdm-extension/syntaxes/cdm16-assembly.tmLanguage.json [hunk 3]
```diff
@@ -86,7 +95,7 @@
         "directives": {
 			"patterns": [{
 				"name": "entity.name.type.cdm16-assembly",
-				"match": "\\b(asect|rsect|dc|ds|tplate|end|end.|macro|mpop|mpush|mend|define)\\b"
+				"match": "\\b(asect|rsect|dc|ds|tplate|end|end.|macro|mpop|mpush|mend|define|db|dw)\\b"
```
Comment (cjvth) [inline vscode-cdm-extension/syntaxes/cdm16-assembly.tmLanguage.json:98]: Add `align` to directives

**Review by cjvth — CHANGES_REQUESTED** [review]

**Review by Intelix8996 — APPROVED** [review]

**Review by cjvth — APPROVED** [review]

**Pull request merged**